src/Microsoft.AspNetCore.Razor.Language/TagHelperMetadata.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    internal static class TagHelperMetadata
+    {
+        public static class Common
```

Comment:
The holder is internal and we don't currently have any plans for more metadata. Should just throw these constants under metadata imo